typings/glamorous.test.tsx

```diff
@@ -0,0 +1,21 @@
+import glamorous from "../";
+
```

Comment:
Note that it doesn't validate whether the declaration file is accurate with respect to the implementation